apps/api/src/common/authority-checker.service.ts

```diff
@@ -117,28 +126,57 @@ export class AuthorityCheckerService {
       throw new NotFoundException(`Project with id ${entity?.id} not found`)
     }
 
+    const projectAccessLevel = project.accessLevel
     // Get the authorities of the user in the workspace with the project
-    const permittedAuthorities = await getCollectiveProjectAuthorities(
-      userId,
-      project,
-      prisma
-    )
+    const permittedAuthorities: Set<Authority> =
```

Comment:
Okay, so for this, we now need to split things up. Here's what needs to be done.

- Rename `permittedAuthorities` to `permittedAuthoritiesForProject`.
- Add another set named `permittedAuthoritiesForWorkspace`. Initialize the value from the `getCollectiveWorkspaceAuthorities` function in `get-collective-workspace-authorities.ts` file.

Reply:
Added both `permittedAuthoritiesForProject` as well as `permittedAuthoritiesForWorkspace`